mount/mount_solaris.go

```diff
@@ -0,0 +1,55 @@
+package mount
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	mountCmd = "/usr/sbin/mount"
+)
+
+func doMount(arg ...string) error {
```

Comment:
i've enhanced the comments here to discuss the issues related to creating mounts on solaris.